security/practical_security.md

```diff
@@ -0,0 +1,5 @@
+So this prompted me to finally set-up 2-factor authentication on all my iphone and bank accounts.
+I had been putting it off for some time but after going through security, I realized
+it was probably in my best interest to get it done to further protect myself. On top of that I 
+used a random password generator to help protect both my email and banking accounts to make
+that more secure. I made sure to store the password in a place that I could get to if I need it but that would be extremely difficult for anyone else to get to.
```

Comment:
Awesome! Two factor and difficult passwords are super helpful in making sure that your information is secure. I recommend setting up a password manager as well. Most have features that will automatically generate secure passwords for you and change the passwords to your accounts automatically for you. It makes it super easy to be secure.